src/components/UI/Overlay/OverlayContent/DocumentTransferMessage.tsx

```diff
@@ -143,6 +146,18 @@ export const MessageNominateBeneficiaryHolderSuccess = () => {
   return <p>Document has been nominated successfully. Please notify holder to execute transfer.</p>;
 };
 
+export const MessageEndorseTransferSuccess = ({ beneficiaryAddress, holderAddress }: MessageProps) => {
+  return (
+    <>
+      <h6>Current Beneficiary</h6>
+      <p>{beneficiaryAddress}</p>
+      <div />
+      <h6>Current Holder</h6>
+      <p>{holderAddress}</p>
```

Comment:
wanna use the address resolver for all potential overlay addresses? 

maybe something like:

```
export const MessageAddressResolved = ({ address }: { address: string }) => {
  const { resolvedIdentifier } = useIdentifierResolver(address);
  return <p>{resolvedIdentifier}</p>;
};
```

then in the messages:

```
export const MessageEndorseTransferSuccess = ({ beneficiaryAddress, holderAddress }: MessageProps) => {
  return (
    <>
      <h6>Current Beneficiary</h6>
      {beneficiaryAddress && <MessageAddressResolved address={beneficiaryAddress} />}
      <div />
      <h6>Current Holder</h6>
      {holderAddress && <MessageAddressResolved address={holderAddress} />}
    </>
  );
};
```

think can do the same with bene and holder successes too

Reply:
ok i'll update for the other message that requires the address too.

Reply:
might wanna rename? got another component called AddressResolver already 😅

Reply:
haha ok didnt notice it.